spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/it/run-arguments-commandline/invoker.properties

```diff
@@ -0,0 +1 @@
+invoker.mavenOpts=-Dspring-boot.run.arguments='--management.endpoints.web.exposure.include=prometheus,info,health,metrics,--spring.profiles.active=foo,bar'
```

Comment:
I think it would be more natural if the arguments were separated by a space rather than a comma.